pandas/tests/indexes/interval/test_interval.py

```diff
@@ -616,6 +624,14 @@ def test_get_indexer_length_one(self, item, closed):
         expected = np.array([0] * len(item), dtype='intp')
         tm.assert_numpy_array_equal(result, expected)
 
+    @pytest.mark.parametrize('index', [
+        pd.interval_range(0, 1),
```

Comment:
I would add here one that is a bit larger as well (like `pd.interval_range(0, 3)`), as 1-element indices can behave a bit special regarding monotonicity / sortedness (as they always are that)